elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/impl/ElasticsearchSimpleFlowStage.scala

```diff
@@ -79,7 +79,7 @@ private[elasticsearch] final class ElasticsearchSimpleFlowStage[T, C](
       log.debug("Posting data to Elasticsearch: {}", json)
 
       if (json.nonEmpty) {
-        val uri = baseUri.withPath(Path(endpoint))
+        val uri = baseUri.withPath(Path(baseUri.path.toString() + endpoint))
```

Comment:
This doesn't look right, `baseUri` should already be there as this attaches to it.

Reply:
if `baseUri` is `https://test.com:9200/elasticsearch` , it is ignoring` /elasticsearch` path.

Reply:
if `baseUri` is` https://test.com:9200/elasticsearch`, it is ignoring` /elasticsearch` path

Reply:
![image](https://github.com/akka/alpakka/assets/9468378/8f821e00-ff1d-4986-ab65-1f3035598274)

Reply:
I believe what you are looking for is `baseUri.withPath(baseUri.path / endpoint)` but can't verify that right now.